yarn.lock

```diff
@@ -4512,7 +4512,7 @@
   resolved "https://registry.yarnpkg.com/@uniswap/redux-multicall/-/redux-multicall-1.1.5.tgz#7c097047d489c1624038c0fbbd3d76dc705bf153"
   integrity sha512-RSMhfuAX2rPimnevvAAiwoyV2bCGTIKhVHEBOLTMF+oVxYcKKe9hCwx/cffY12t/usXWHlEJ//V7JoxTKI1Lyg==
 
-"@uniswap/router-sdk@^1.0.3", "@uniswap/router-sdk@^1.0.5":
+"@uniswap/router-sdk@^1.0.5":
```

Comment:
both @uniswap/router-sdk and @uniswap/v3-sdk should be de-duped.
You should be able to do so by running

    npx yarn-deduplicate --packages @uniswap/router-sdk @uniswap/v3-sdk
    
Could you do so and ping me so I can re-check the lockfile?